packages/js/client/src/Web3ApiClient.ts

```diff
@@ -46,10 +53,18 @@ export class Web3ApiClient implements Client {
 
     // Add all default redirects (IPFS, ETH, ENS)
     this._config.redirects.push(...getDefaultRedirects());
+    this._invokeContextMap.set(DEFAULT_CONTEXT_ID, {
```

Comment:
This is a bit of a hack because we use loadWeb3Api directly inside SchemaCompose.

https://github.com/Web3-API/monorepo/blob/5c1fbddf253783709c923a6247e74b1769e694fb/packages/cli/src/lib/SchemaComposer.ts#L113

Reply:
This has been removed since we are not using it anymore in the CLI